src/bin/analyzer.js

```diff
@@ -81,10 +86,29 @@ let {
   logLevel,
   open: openBrowser,
   exclude: excludeAssets,
+  info,
   args: [bundleStatsFile, bundleDir]
 } = program;
 const logger = new Logger(logLevel);
 
+const packageInfo = {
+  node: 'node -v',
+  npm: 'npm -v',
+  yarn: 'yarn -v',
+  webpack: 'webpack-cli -v',
+  webpackBundleAnalyzer: 'webpack-bundle-analyzer -V'
+};
+
+if (info) {
+  console.log(' Environment info:-');
+  try {
+    Object.keys(packageInfo).forEach(pkg => console.log(` ${pkg} - ${execSync(packageInfo[pkg])}`));
```

Comment:
It's better to use `exec` instead of `execSync` to run commands in parallel to speed up execution.